packages/components/file-card/src/file-card.css

```diff
@@ -1,9 +1,20 @@
 :host {
 	display: block;
+	margin-bottom: var(--ts-unit-half);
 }
 
-.action-icon,
-.file-icon-wrapper .file-icon {
+.file-icon-wrapper {
+	position: relative;
+	height: 40px;
```

Comment:
It is not good to use numbers instead of `--ts-unit`

Reply:
Yeah you are right! 👍  I just copied from abstract!